cpp/src/io/parquet/page_decode.cuh

```diff
@@ -423,27 +430,32 @@ inline __device__ int gpuDecodeRleBooleans(page_state_s* s, state_buf* sb, int t
   return pos;
 }
 
+/**
+ * @brief Indicates if the string descriptor initializer kernel is only calculating sizes
+ */
+enum class is_calc_sizes_only : bool { NO = false, YES = true };
+
 /**
  * @brief Parses the length and position of strings and returns total length of all strings
  * processed
  *
  * @param[in,out] s Page state input/output
  * @param[out] sb Page state buffer output
  * @param[in] target_pos Target output position
- * @param[in] g Cooperative group (thread block or tile)
- * @tparam sizes_only True if only sizes are to be calculated
+ * @param[in] group Cooperative group (thread block or tile)
+ * @tparam sizes_only Indicates if only sizes are to be calculated
  * @tparam state_buf Typename of the `state_buf` (usually inferred)
  * @tparam thread_group Typename of the cooperative group (inferred)
  *
  * @return Total length of strings processed
  */
-template <bool sizes_only, typename state_buf, typename thread_group>
+template <is_calc_sizes_only sizes_only, typename state_buf, typename thread_group>
 __device__ size_type gpuInitStringDescriptors(page_state_s* s,
                                               [[maybe_unused]] state_buf* sb,
                                               int target_pos,
-                                              thread_group const& g)
+                                              thread_group const& group)
 {
-  int const t         = g.thread_rank();
+  int const tid       = group.thread_rank();
```

Comment:
why this change?

Reply:
Just personally don't like single letter variable names. Nothing else 😄

Reply:
we pretty consistently use `t` in io (1440 vs 24). I don't mind `tid`, but we probably want to make sure we're consistent, given that neither name is descriptive. I immediately assumed that `tid` has a different meaning, and that's why it's been renamed.

Reply:
Let me revert this to `t`